docs/src/plasma/qtplasmac.adoc

```diff
@@ -901,22 +901,22 @@ For more information, see the <<qt_pm_comms,PowerMax Communications>> section.
 
 [width="100%",cols="4,16",options="header"]
 |===
-|Name  |Description
-|FLOAT |Indicates that the float switch is activated.
-|OHMIC |Indicates that the probe has sensed the material.
-|BREAK |Indicates that the torch breakaway sensor is activated.
+|Name   |Description
+m|FLOAT |Indicates that the float switch is activated.
+m|OHMIC |Indicates that the probe has sensed the material.
+m|BREAK |Indicates that the torch breakaway sensor is activated.
```

Comment:
IMHO the monospace font does not really fit here. The names in the left column are just the button names.

Also for the following "m"s.
Furthermore a description could be added what the tables represent and the table header could be named more specific. Like "button name instead if just "name".